pages/e-lab.tsx

```diff
@@ -261,3 +268,32 @@ function Hero() {
     </section>
   );
 }
+
+function SliderCard({ imgSrc, name, text }) {
+  return (
+      <div className="relative h-full w-full min-w-sm rounded-lg grayscale saturate-200 overflow-hidden">
+        <Image src={imgSrc} alt={name} fill objectFit="cover"/>
+        <div className="h-3/8 absolute bottom-0 w-full bg-black bg-opacity-80 p-2 text-white backdrop-blur">
+          <h2 className="font-bold">{name}</h2>
+          <p>{text}</p>
+        </div>
+      </div>
+  );
+}
+
+function SnapSlider({ data }) {
+  return (
+      <div className="snap-x flex overflow-x-auto scrollbar-hide space-x-8 pb-4">
+        {data.map((item, index) => (
+            <div className="snap-center inline-flex min-w-[220px] min-h-[220px] align-middle" key={index}>
```

Comment:
using index as key should be last resort if nothing else is possible
```suggestion
            <div className="snap-center inline-flex min-w-[220px] min-h-[220px] align-middle" key={`${item.name}-${item.text}`}>
```